README.md

```diff
@@ -115,6 +115,28 @@ namespace SampleApp.Activities
 }
 ```
 
+## INotifyDataErrorInfo Support
+
+For those platforms which support the `INotifyDataErrorInfo` interface, ReactiveUI.Validation provides a helper base class named `ReactiveValidationObject<TViewModel>`. The helper class implements both the `IValidatableViewModel` interface and the `INotifyDataErrorInfo` interface. It listens to any changes in the `ValidationContext` and invokes `INotifyDataErrorInfo` events. 
+
+```cs
+public class IndeiTestViewModel : ReactiveValidationObject<IndeiTestViewModel>
```

Comment:
What's Indei btw

Reply:
`INotifyDataErrorInfo` abbreviation, seen in Avalonia https://github.com/AvaloniaUI/Avalonia/blob/d6578eba5089e267d9701a6b7380a5830fbe2465/samples/BindingDemo/ViewModels/IndeiErrorViewModel.cs and elsewhere